spec/system/admin/admin_orders_csv_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'tempfile'
+
+RSpec.describe 'Admin orders export csv' do
+  let!(:admin) { create(:admin_user) }
+  let!(:order1) { create(:order) }
+  let!(:order2) { create(:order) }
+
+  before do
+    login_as admin, scope: :admin_user
+    visit root_path
+  end
+
+  it 'exports csv with all order fields' do
+    click_link 'Admin'
+    click_link 'Orders'
+    click_on 'Download all orders as CSV'
+
+    # write new Tempfile
+    file = Tempfile.new
+    file.write(page.body)
+    file.rewind
+    csv = file.read
+    file.close
+    file.unlink
```

Comment:
I know i could use here `csv = CSV.new(page.body).read`, but i wanted to learn Tempfile gem here.

Reply:
but isn't the tempfile part of stdlib in Ruby? https://ruby-doc.org/stdlib-2.5.3/libdoc/tempfile/rdoc/Tempfile.html